frontend/src/components/dashboard/incidentsHomePage/IncidentsHomePage.js

```diff
@@ -110,20 +111,31 @@ export class IncidentsHomePage extends Component {
     this.state = {
       incidentsLoaded: false,
       params: {
+        page: Strings.DEFAULT_PAGE,
+        per_page: Strings.DEFAULT_PER_PAGE,
         region: formatParamStringToArray(this.params.region),
         major: this.params.major,
         search: this.params.search,
         year: this.params.year,
         incident_status: this.params.incident_status,
         codes: this.params.codes,
         determination: this.params.determination,
+        ...this.params,
       },
     };
   }
 
   componentDidMount() {
     const params = this.props.location.search;
-    this.renderDataFromURL(params);
+    if (params) {
+      this.renderDataFromURL(params);
+    } else {
+      const defaultParams = {
```

Comment:
Option 1) 
if there are no 'params' in the URL - then 'page' and 'per_page' are already defined in the state with their default when the page loads, so this latter step is not necessary

Option 2) 
the URL contains the params - if 'ComponentWillReceiveProps' is handling the params correctly, it should be setting the new values for 'page' and 'per_page'

Reply:
@thegentlemanphysicist